federatorai/manifest.json

```diff
@@ -0,0 +1,28 @@
+{
+  "display_name": "federatorai",
+  "maintainer": "support@prophetstor.com",
+  "manifest_version": "1.0.0",
+  "name": "federatorai",
+  "metric_prefix": "federatorai.",
+  "metric_to_check": "",
```

Comment:
Ideally there should be a metric to check. It should be a metric that is always emitted and it's used is to show on the integration tile that the integration is functioning correctly

Reply:
thanks, i have added "federatorai.integration.status" to "metric_to_check"